python/main.py

```diff
@@ -40,38 +41,76 @@ def add_item(name: str = Form(...), category: str = Form(...), image: str = Form
             ext = ".jpg"
             hash_img = sha256 + ext
             os.rename(copy, f"./images/{hash_img}")
-    # error handling
+    # Error handling (no image)
     else:
         logger.info("Image not found")
         hash_img = "..."
 
+    #save info into a db file
+    path = '../db/items.db'
+    if os.path.isfile(path) == False:
+        connect = sqlite3.connect(path)
+        cursor = connect.cursor()
+        cursor.execute("CREATE TABLE category(id integer PRIMARY KEY, name string unique)")
+        cursor.execute("CREATE TABLE items(id integer PRIMARY KEY, name string, category_id integer, image_name string, FOREIGN KEY(category_id) REFERENCES category(id))")
+    connect = sqlite3.connect(path)
+    cursor = connect.cursor()
+    cursor.execute("PRAGMA foreign_keys=True")
+    cursor.execute("INSERT INTO category(name) values(?) ON CONFLICT(name) DO NOTHING", (category,))
+    cursor.execute("SELECT id FROM category WHERE name=?", (category,))
+    log = cursor.fetchone()
+    i = int(log[0])
+    cursor.execute("INSERT INTO items(name, category_id, image_name) values(?, ?, ?)", (name, i, hash_img))
+    connect.commit()
+    cursor.close()
+    connect.close()
+    return ({"message": f"item received: {name}"})
```

Comment:
Pythonのreturn時には`(` `)`は特段不要です。
```suggestion
    return {"message": f"item received: {name}"}
```